src/sentry/lang/javascript/processor.py

```diff
@@ -972,21 +961,25 @@ def process_frame(self, processable_frame, processing_task):
                     }
                 )
 
-            # persist the token so that we can find it later
-            processable_frame.data["token"] = token
-
             # Store original data in annotation
             new_frame["data"] = dict(frame.get("data") or {}, sourcemap=sourcemap_label)
 
             sourcemap_applied = True
 
             if token is not None:
-                abs_path = non_standard_url_join(sourcemap_url, token.src)
+                if token.src is not None:
```

Comment:
What exactly did the old code do in case the `source` is `null`?

Reply:
It used `sourcemap_url` as the `abs_path`, which was incorrect and our tests for this were also incorrect.